app/assets/javascripts/map.js

```diff
@@ -66,6 +67,21 @@ class Map {
     return new google.maps.Geocoder();
   }
 
+  initAutocomplete(){
+    const defaultBoundaries = new google.maps.LatLngBounds(
+      new google.maps.LatLng(...constants.DEFAULT_BOUNDARIES.BOTTOM),
+      new google.maps.LatLng(...constants.DEFAULT_BOUNDARIES.TOP),
+    );
+    const options= {
+      bounds: defaultBoundaries,
+      types: ['address'],
+      componentRestrictions: {country: 'ca'}
```

Comment:
https://developers.google.com/maps/documentation/javascript/places-autocomplete#add_autocomplete

> `strictBounds` is a boolean specifying whether the API must return only those places that are strictly within the region defined by the given bounds. The API does not return results outside this region even if they match the user input.

Reply:
I didn't want to use this in case they want to branch out to other areas other thanthe tricities

Reply:
can't have your cake and eat it too